libs/ui/src/lib/components/channeling/booking/details.tsx

```diff
@@ -1,11 +1,18 @@
+import { FC } from 'react';
 import { Card, Text, Group, createStyles, Center } from '@mantine/core';
 import { Clock } from 'tabler-icons-react';
 
-const mockdata_name = [{ label: 'Danial Smith' }];
-const mockdata_gend = [{ label: 'Male' }];
-const mockdata = [{ label: '16.30', icon: Clock }];
+export interface CardProps {
+  label_name?: string;
+  lable_gend?: string;
+  label_time?: string;
+}
 
-export const ChannelDetailCard = ({ ...props }) => {
+interface ChannelDetailCardProps {
+  data: CardProps[];
+}
+
+export const ChannelDetailCard: FC<ChannelDetailCardProps> = ({ data }) => {
```

Comment:
You've forgot to add these missing props in other places where this component is imported